deps/chakrashim/src/v8json.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright Microsoft. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and / or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+
+#include "v8.h"
+#include "jsrtutils.h"
+
+namespace v8 {
+    Local<Value> JSON::Parse(Local<String> json_string) {
+        jsrt::IsolateShim* iso = jsrt::IsolateShim::GetCurrent();
+        jsrt::ContextShim* contextShim = iso->GetCurrentContextShim();
+
+        JsValueRef jsonParseFunction = contextShim->GetjsonParseFunction();
+        JsValueRef obj;
+        
+        if (jsrt::CallFunction(jsonParseFunction, (JsValueRef)*json_string,
+            &obj) != JsNoError) {
```

Comment:
Made this change here but the other call to CallFunction which has arguments spread on two lines is not lined-up because it would put that line past the limit. Doesn't seem like there's a hard policy on this?